package.json

```diff
@@ -4,14 +4,15 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "npx @11ty/eleventy",
```

Comment:
Probs don't need test, unless you want it, but `build` is nice when you want to make sure a build succeeds but don't need to watch. And the build command you set in cloudflare should match this!

Reply:
Yep, I think it's good to have a build command set up, just hadn't bothered yet 😆